src/plugins/discover/public/application/components/chart/download_csv_button/download_csv_button.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useMemo } from 'react';
+import { EuiButtonEmpty } from '@elastic/eui';
+import { FormattedMessage } from '@osd/i18n/react';
+import Papa from 'papaparse';
+// @ts-ignore
+import { saveAs } from '@elastic/filesaver';
```

Comment:
Can we choose an npm package that's being actively maintained? The github repo for this package is archived I believe, and the latest version is from 7 years ago. Lack of maintenance of 3rd party lib cause us trouble for things like CVE. Not to mention, moving forward beyond 3.0, we definitely want to get rid of all @elastic dependencies
https://www.npmjs.com/package/@elastic/filesaver

Reply:
@zhongnansu Hey! I appreciate this comment, as I was really hesitant on using this code too, but decided to keep with the 'norm' of the other codebases (this is used in other components).

But now that I'm not the only one thinking it, I will happily not use this. Its actually completely unnecessary IMO, it can be done with basic JS without any libraries

Reply:
got it. I am OK with it, since other legacy code also import this dependency. Not sure if we have a meta issue of removing all @elasctic dependency, definitely worth opening one to track IMO.

Reply:
Thats a good idea, i'll leave it in for now but I have opened an issue to remove at least @elastic/filesaver globally from our codebase: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/9341